catalog/ui/search-ui/search-endpoint/pom.xml

```diff
@@ -388,9 +388,29 @@
                         </Private-Package>
                         <Export-Package/>
                         <Import-Package>
+                            ddf.action,
+                            ddf.catalog,
```

Comment:
By specifying this don't you import ddf.catalog.filter, operation, source etc? Same question with ddf.security